types/bad-words/bad-words-tests.ts

```diff
@@ -0,0 +1,18 @@
+import Filter = require('bad-words');
+
+new Filter();
+new Filter({
+    emptyList: true,
+    list: [],
+    exclude: [],
+    placeHolder: 'a',
+    regex: /a/,
+    replaceRegex: /a/,
+    splitRegex: /a/,
+});
+const f = new Filter();
+f.addWords('a', 'b', 'c');
```

Comment:
i wasn't sure how to check that these return void since `const add: undefined | void` both didn't work

Reply:
```suggestion
// $ExpectType void
f.addWords('a', 'b', 'c');
```

Reply:
awesome, thanks!